### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by Resxt**
Changed "balados" (not a real word) to "podcasts"
Added "hideAudiobooks" translation

### CODE REVIEW

**Comment by theRealPadster** [comment]
When I google "balados", it seems to mean podcasts? 
image

**Comment by theRealPadster** [comment]
If it's a Canadian/Quebec French thing, can you add an override to a separate locale file? Like how we have "en" and "en-US"? They have different languages in Spotify. 
image

**Comment by Resxt** [comment]
I don't speak or understand anything canadian so no, I would mess it up.

My opinion is that they still speak french and understand it. Metropolitan french people don't understand canadian but canadians understand "regular french" because their language is based on it.

The words used for this translation are very simple french and they will understand it. Even more since "podcasts" is widely known and since english is like second nature in a lot of places there

**Comment by theRealPadster** [comment]
You don't need to include every string, I just meant to move the current "balados" ones to a French Canadian file, and then we update the main fr.json.

File: src/locales/fr-CA.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "translation": {
+    "menu": {
+      "title": "Masquer les balados",
+      "enabled": "Activé",
+      "aggressiveMode": "Mode agressif",
+      "hideAudiobooks": "Masquer les livres audio"
+    },
+    "search": {}
```
Comment (theRealPadster) [inline src/locales/fr-CA.json:9]:
```suggestion
      "title": "Masquer les balados"
    }
```

**Review by theRealPadster — CHANGES_REQUESTED** [review]
We just need the changed keys. It will default to the regular "fr.json" version for anything missing.

**Review by theRealPadster — APPROVED** [review]

**Pull request merged**